src/services/ReviewService.js

```diff
@@ -189,14 +191,15 @@ function * _updateReview (authUser, reviewId, entity) {
       'id': reviewId
     },
     UpdateExpression: `set score = :s, scoreCardId = :sc, submissionId = :su,
-                        typeId = :t, reviewerId = :r,
+                        typeId = :t, reviewerId = :r, status = :st
```

Comment:
@meshde Kindly add a comma here - I got a syntax error when making the api call:

```
{
    "message": "Invalid UpdateExpression: Syntax error; token: \"updated\", near: \":st\n                        updated =\""
}
```

Reply:
@meshde Also - `status` is a reserved keyword in dynamodb. You cannot use it directly. You will have to use `ExpressionAttributeNames`

Reply:
@callmekatootie Have made the required fixes.